### PR TITLE
MAUI-3988 : Updated MAUI Maps UG and resized the images.

### PR DESCRIPTION
**Pull request by Shalini-Ashokan**
Hi @vigneshrameshsync,

I have completed the review changes and resized the UG images. Could you please check it?

Regards,
Shalini Ashokan.

### CODE REVIEW

File: MAUI/Maps/custom-shape.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
+---
+layout: post
+title: About .NET MAUI Maps control | Syncfusion 
+description: Learn here all about the custom shapes feature of Syncfusion .NET MAUI Maps (SfMaps) control, its features, and more.
+platform: MAUI
+control: SfMaps
+documentation: ug
+---
+
+# Custom shapes in .NET MAUI Maps (SfMaps)
```
Comment (vigneshrameshsync) [inline MAUI/Maps/custom-shape.md:10]: This file is not included in toc file.

Comment (Shalini-Ashokan) [reply MAUI/Maps/custom-shape.md:10]: Addd in toc file

File: MAUI/Maps/custom-shape.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+{% endhighlight %}
+
+{% highlight c# %}
+
+public MainPage()
+{
+    InitializeComponent();
+    SfMaps map = new SfMaps();
+    MapShapeLayer layer = new MapShapeLayer();
+    layer.ShapesSource = MapSource.FromResource("MyProject.buildings.json");
```
Comment (vigneshrameshsync) [inline MAUI/Maps/custom-shape.md:36]: buildings -> floor_planning

File: MAUI/Maps/custom-shape.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+    layer.Geometry = MapGeometryType.Points;
+    map.Layer = layer;
+    this.Content = map;
+}
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![maps basic view](images/custom-shape/maps_cartesian_view.png)
```
Comment (vigneshrameshsync) [inline MAUI/Maps/custom-shape.md:48]: Change alt text as **.NET MAUI Maps with custom shapes**

**Review by vigneshrameshsync — APPROVED** [review]
Fine.

**Pull request merged**